tests/atoms/gaia_apps.js

```diff
@@ -372,13 +372,13 @@ var GaiaApps = {
          .query('AppWindowManager.getActiveWindow').getTopMostWindow();
     }
 
-    let origin = app.origin;
-    console.log('app with origin \'' + origin + '\' is displayed');
+    console.log('app with origin and manifest url \'' + app.origin + app.manifest_url + '\' is displayed');
```

Comment:
I'd put `app.origin + ', ' + app.manifest_url`. Otherwise you'll end up with a linelikethis.